src/ophys_etl/pipelines/brain_observatory/scripts/run_mesoscope_splitting.py

```diff
@@ -58,26 +60,56 @@ def convert_column(input_tif, session_storage, experiment_info, **h5_opts):
     return conversion_output(mt.volume_views[0], filename, experiment_info)
 
 
-def split_z(input_tif, experiment_info, **h5_opts):
-    directory = experiment_info["storage_directory"]
-    eid = experiment_info["experiment_id"]
+def split_z(input_tif: MesoscopeTiff,
+            experiments: List[Dict],
+            testing=False,
+            **h5_opts) -> Tuple[Dict, Dict]:
+    """Takes a z_stack file from a Mesoscope ophys session and a list
+    of the experiments performed during that session and splits the data
+    into a z_stack file (as a .h5) for each individual experiment.
+
+    Parameters
+    ----------
+    input_tif : MesoscopeTiff
+        MesoscopeTiff object containing the timeseries data that
+        will be split by experiment
+
+    experiments : List[Dict]
+        A list of dictionaries, each containing information about
+        the experiments performed during the ophys session
```

Comment:
please add something like: "each dictionary is described by SchemaX (whichever one it is) in repo.path.to.schemas"